cmd/util/cmd/read-ledger-wals/main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "github.com/onflow/flow-go/cmd/util/cmd/read-ledger-wals/cmd"
+
+func main() {
+	cmd.Execute()
```

Comment:
does it make sense to have it as it's own cmd? or should we add it to general utils?

Reply:
I moved it to the read-exec-state as a subcommand.

Reply:
applied at bb3bcac